packages/next-templates/src/app/mijn-omgeving/inloggen/page.tsx

```diff
@@ -0,0 +1,107 @@
+'use client';
+
+import {
+  Article,
+  Button,
+  ButtonGroup,
+  Heading1,
+  Heading2,
+  Icon,
+  Link,
+  Page,
+  PageContent,
+  Paragraph,
+} from '@utrecht/component-library-react';
+import { UtrechtDigidLogo } from '@utrecht/web-component-library-react';
+import { ExampleHeaderFunnelWmebv } from '@/components/wmebv/Header/ExampleHeaderFunnelWmebv';
+import { ExampleFooterWmebv } from '@/components/wmebv/Footer/ExampleFooterWmebv';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { Fieldset, FieldsetLegend, FormField, FormLabel, RadioButton } from '@utrecht/component-library-react';
+import BacklinkIcon from '@/app/styling/assets/backlink-icon.svg';
+import { IconArrowLeft } from '@tabler/icons-react';
+import '@/app/styling/css/wmebv.css';
+import '@/app/styling/css/detail-page.css';
+import '@utrecht/design-tokens/dist/index.css';
+import { useEffect } from 'react';
+
+export default function home() {
+  const stepLabel = 'Vraag aan de gemeente';
```

Comment:
Hoort hier 'vraag aan de gemeente' te staan? Volgens mij is dat van een andere pagina dan voor mijn omgeving maar not sure.

Reply:
Er is geen voorbeeld die ik volgde. Er stond in de pb inlog formulier net in contact formulier. Daarvan heb ik het gelaten

Reply:
dit is de title die je in de <HEAD> ziet zitten die kan je gewoon her noemen `<title>Vraag aan de gemeente - gemeente Voorbeeld</title>` als voorbeeld kan je er neer zetten inloggen mijn omgeving

Reply:
Gedaan